build.gradle

```diff
@@ -56,10 +56,13 @@ subprojects {
 		gson
 		powertools
 		mockito
+		bouncycastle
 	}
 
 	dependencies {
 
+		bouncycastle 'org.bouncycastle:bcpkix-jdk15on:1.70'
```

Comment:
Is bouncy castle actually needed? I can't see it referenced in any of the code (unless it is abstracted away by Nimbus)

Reply:
It isn't needed! Removed